README.md

```diff
@@ -1 +1,37 @@
-# spec
+# OpenEvents
+
+Events are everywhere.  However, event publishers tend to describe events differently.
```

Comment:
Do we need to be explicit about whether we're talking about events as they appear "on the wire" vs "as presented to the function" or both?  This is probably related to my "usecases" comment below - meaning I think our target usecases should make it clear where the level of interop is that we're shooting for.

Reply:
Yes, but I don't think that distinction needs to be made in this line.